src/openms/source/METADATA/ProteinIdentification.cpp

```diff
@@ -81,7 +81,8 @@ namespace OpenMS
     enzyme(UNKNOWN_ENZYME),
     missed_cleavages(0),
     peak_mass_tolerance(0.0),
-    precursor_tolerance(0.0)
+    precursor_tolerance(0.0),
+    digestion_enzyme("unknown_enzyme","")
```

Comment:
But this will be copied many times in memory, right?

sent from my smartphone -- you may keep any typos you find.
On Apr 15, 2015 11:36 PM, "Timo Sachsenberg" <notifications@github.com>
wrote:

> In src/openms/source/METADATA/ProteinIdentification.cpp
> <https://github.com/OpenMS/OpenMS/pull/1288#discussion_r28465571>:
>
> > @@ -81,7 +81,8 @@ namespace OpenMS
> >      enzyme(UNKNOWN_ENZYME),
> >      missed_cleavages(0),
> >      peak_mass_tolerance(0.0),
> > -    precursor_tolerance(0.0)
> > +    precursor_tolerance(0.0),
> > +    digestion_enzyme("unknown_enzyme","")
>
> I think it is a good idea to store an actual enzyme (no pointer).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenMS/OpenMS/pull/1288/files#r28465571>.
>